### ISSUE TITLE
Integrate Courts-db into Courtlistener

### ISSUE THREAD

**Issue by flooie** [freelawproject/courtlistener#1230]
We have a brand new project courts-db that we should start using to identify courts in CL.

**Comment by quevon24** [comment]
@mlissner @flooie I've inspected courtlistener source code to try to find where can we use courts-db, but couldn't find a place where we could implement it or replace something with courts-db, do you have any idea where we should use the library?

**Comment by mlissner** [comment]
I think the idea is to remove this file: https://github.com/freelawproject/courtlistener/blob/4ca2d63f9432603a6e33555bf03677580dd596f7/cl/corpus_importer/court_regexes.py
https://github.com/troglodite2/courtlistener/blob/4ca2d63f9432603a6e33555bf03677580dd596f7/cl/corpus_importer/court_regexes.py

I haven't looked at how it's used. Want to report back on that, and then we can decide if we want to replace functionality or just remove it?

**Comment by quevon24** [comment]
> Technically, we already started using it ...
> 
> `cl/corpus_importer/management/commands/harvard_opinions.py`

Yes, besides harvard opinion imported, is there anywhere else we should use the library? I used it in lexis and westlaw citation importer

**Comment by flooie** [comment]
@quevon24 I think @mlissner pointed out that we should see if we can remove the giant regexes in the corpus importer

**Comment by mlissner** [comment]
@flooie, can we analyze this and maybe do it. Seems like an easy win?
